external/wakaama/examples/client/lwm2mclient.c

```diff
@@ -1259,21 +1267,25 @@ int lwm2m_client_main(int argc, char *argv[])
     if (NULL == objArray[8])
     {
         fprintf(stderr, "Failed to create Access Control object\r\n");
+		close(data.sock);
         return -1;
     }
```

Comment:
Please, check final source codes for external/wakaama/examples/client/lwm2mclient.c. Do not have these codes. We have changed back wakama to it's original state. I mean original opensource.